tools/memleak_example: fix typos

ekyooo opened the pull request.

yonghong-song commented:
Something is wrong in your patch. The libbpf submodule is not the same as the current one. Please sync with the latest master branch and change only memleak_example and resubmit.

ekyooo commented:
> Something is wrong in your patch. The libbpf submodule is not the same as the current one. Please sync with the latest master branch and change only memleak_example and resubmit.

Oops. sorry. I fixed it and rebased to the latest master branch.
Thank you.
